### DISCUSSION TITLE
PDF COMPRESSION TECHNIQUES

### DISCUSSION THREAD

**Discussion by ACCAWSENSE** [pdf-association/pdf-issues#498] [General]
Dear Members , 

Request help on Extreme Compression of PDF by 70% to 90% , using very Optimization techniques . 

ALso kindly suggest any Open Source PDF Libraries with all the Tools of PDF eg. PDF CONVERTOR , PDF COMPRESSOR , SPLIT , MERGE , ZIP , PASSOWORD PROTECT , EDIT , SIGN ETC ETC. 

Regds

Team Awsense

**Comment by petervwyatt** [comment]
This question is better suited to software forums such as Stack Overflow or social media channels, where vendors and implementors can respond.

**Comment by ACCAWSENSE** [comment]
Ok thx for your reply …

 

Regds

 

Aakash chhabria
